operator/controllers/kyma_controller.go

```diff
@@ -220,6 +208,30 @@ func (r *KymaReconciler) ReconcileFromConfigMap(ctx context.Context, req ctrl.Re
 	return nil
 }
 
+func (r *KymaReconciler) GetTemplatedComponent(templateName, componentTemplate string, release *KymaReleaseProgressionInfo) (map[string]interface{}, error) {
```

Comment:
I am not sure if we should template components. It can become fragile. You provide the same information in labels.